source/common/http/filter/fault_filter.cc

```diff
@@ -0,0 +1,159 @@
+#include "fault_filter.h"
+
+#include "envoy/event/timer.h"
+#include "envoy/http/codes.h"
+#include "envoy/http/header_map.h"
+#include "envoy/stats/stats.h"
+
+#include "common/common/assert.h"
+#include "common/common/empty_string.h"
+#include "common/http/codes.h"
+#include "common/http/header_map_impl.h"
+#include "common/http/headers.h"
+
+namespace Http {
+
+FaultFilterConfig::FaultFilterConfig(const Json::Object& json_config, Runtime::Loader& runtime,
+                                     const std::string& stat_prefix, Stats::Store& stats)
+    : runtime_(runtime), stats_{ALL_FAULT_FILTER_STATS(POOL_COUNTER_PREFIX(stats, stat_prefix))} {
+
+  abort_percent_ = 0UL;
```

Comment:
nit: Just initialize all 4 of this in the header file by default initialization. E.g.,
uint64_t abort_percent_{};